docs/enforcement/inline.asciidoc

```diff
@@ -0,0 +1,255 @@
+-A postrouting-int-inline-if -m set --match-set PF-iL2_ID2_192.168.2.0 dst -j CLASSIFY --set-class 1:2
+-A postrouting-int-inline-if -m set --match-set PF-iL2_ID3_192.168.2.0 src -j CLASSIFY --set-class 1:3
+-A postrouting-int-inline-if -m set --match-set PF-iL2_ID3_192.168.2.0 dst -j CLASSIFY --set-class 1:3
+----
+
+So here the role id 1 will have the class 1:1.
+
+Configure Traffic Shaping
++++++++++++++++++++++++++
+
+Here 2 examples of tc rules, the first one will apply an upload/download of:
+1mb/1mb on role id 1
+2mb/2mb on role id 2
+3mb/3mb on role id 3
+
+----
+tc qdisc del dev eth0 root
+tc qdisc add dev eth0 root handle 1:0 htb default 1
+
+tc class add dev ens160 parent 1:0 classid 1:1 htb rate 1mbit ceil 1mbit
+tc class add dev ens160 parent 1:0 classid 1:2 htb rate 2mbit ceil 2mbit
+tc class add dev ens160 parent 1:0 classid 1:3 htb rate 3mbit ceil 3mbit
+tc qdisc add dev ens160 parent 1:1 sfq
+tc qdisc add dev ens160 parent 1:2 sfq
+tc qdisc add dev ens160 parent 1:3 sfq
+
+tc qdisc del dev ens192 root
+tc qdisc add dev ens192 root handle 1:0 htb default 1
+
+tc class add dev ens192 parent 1:0 classid 1:1 htb rate 1mbit ceil 1mbit
+tc class add dev ens192 parent 1:0 classid 1:2 htb rate 2mbit ceil 2mbit
+tc class add dev ens192 parent 1:0 classid 1:3 htb rate 3mbit ceil 3mbit
+tc qdisc add dev ens192 parent 1:1 sfq
+tc qdisc add dev ens192 parent 1:2 sfq
+tc qdisc add dev ens192 parent 1:3 sfq
+----
+
+The second one will apply an upload/download of:
+1mb/10mb on role id 1
+2mb/20mb on role id 2
+3mb/30mb on role id 3
+
+----
+tc qdisc del dev ens160 root
+tc qdisc add dev ens160 root handle 1:0 htb default 1
+
+tc class add dev ens160 parent 1:0 classid 1:1 htb rate 1mbit ceil 1mbit
+tc class add dev ens160 parent 1:0 classid 1:2 htb rate 2mbit ceil 2mbit
+tc class add dev ens160 parent 1:0 classid 1:3 htb rate 3mbit ceil 3mbit
+tc qdisc add dev ens160 parent 1:1 sfq
+tc qdisc add dev ens160 parent 1:2 sfq
+tc qdisc add dev ens160 parent 1:3 sfq
+
+tc qdisc del dev ens192 root
+tc qdisc add dev ens192 root handle 1:0 htb default 1
+
+tc class add dev ens192 parent 1:0 classid 1:1 htb rate 10mbit ceil 10mbit
+tc class add dev ens192 parent 1:0 classid 1:2 htb rate 20mbit ceil 20mbit
+tc class add dev ens192 parent 1:0 classid 1:3 htb rate 30mbit ceil 30mbit
+tc qdisc add dev ens192 parent 1:1 sfq
+tc qdisc add dev ens192 parent 1:2 sfq
+tc qdisc add dev ens192 parent 1:3 sfq
+----
+
+Testing the Inline Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can now test the registration process. In order to do so:
+
+* connect an unregistered device into the switch
+* make sure PacketFence provides an IP address to the device. Look into the following log file: /usr/local/pf/logs/packetfence.log or verify on the computer you obtain an ip the right subnet range
+
+From the computer:
+
+* open a web browser
+* try to connect to a HTTP site (Not HTTPS, eg. http://www.google.com)
```

Comment:
I'd use packetfence.org since google has pretty tight HSTS headers that may get the browser to go straight into HTTPS